src/atomate2/vasp/flows/defect.py

```diff
@@ -43,6 +43,18 @@
 
 @dataclass
 class ConfigurationCoordinateMaker(BaseCCDMaker):
+    """Maker to calculate the configuration coordinate diagram.
+
+    Parameters
+    ----------
+    name: str
+        The name of the flow created by this maker.
+    relax_maker: BaseVaspMaker
+        A maker to perform the relaxation of the defect.
+    static_maker: BaseVaspMaker
+        A maker to perform the static calculation of the defect.
+    """
+
```

Comment:
@utf This public class had no doc string. I added one but not sure it's correct. Could you check?

Reply:
Looks great. Thanks